jbmc/unit/java_bytecode/java_bytecode_convert_method/convert_invoke_dynamic.cpp

```diff
@@ -188,7 +188,7 @@ void validate_local_variable_lambda_assignment(
 SCENARIO(
   "Converting invokedynamic with a local lambda",
   "[core]"
-  "[lambdas][java_bytecode][java_bytecode_convert_method][!mayfail]")
+  "[lambdas][java_bytecode][java_bytecode_convert_method]" XFAIL)
```

Comment:
this should be `[!shouldfail]` (it’s `shouldfail` in the macro), and I still dont think this should be a macro.

Reply:
Fixed in later commit